### PR TITLE
adjust config for fight tips

### PR DESCRIPTION
**Pull request by rexatlanti**
Idk if I need to create 19 dummy files called "fight-tips.md"

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -1765,6 +1890,31 @@ collections:
             fields:
               - {label: "Date", name: "date", widget: "datetime", date_format: "YYYY-MM-DD", time_format: false}
               - {label: "Message", name: "message", widget: "string"}
+      - label: "Fight Tips"
+        name: "fight-tips"
+        file: "content/jobs/chealers/scholar/fight-tips.md"
```
Comment (22a) [inline static/admin/config.yml:1895]:
```suggestion
        file: "content/jobs/healers/scholar/fight-tips.md"
```

**Comment by 22a** [comment]
> Idk if I need to create 19 dummy files called "fight-tips.md"

It looks like we have a markdown file for all other content markdown files referenced in this yaml config.

By the looks of https://github.com/The-Balance-FFXIV/balance-static/pull/1384 and other PRs for these content directories these files are never manually generated but rather come from the Netlify CMS automation - which would make me think we'd need to add blank/placeholder content entities in the CMS for these pages and let that get propagated back to the repo. I don't know what the expected behaviour is if you reference a file here and it doesn't exist.

Perhaps @lailleo knows what the process is for scaffolding out new sites that reference content originating from Netlify?

---

Other than that, this config looks correct, equivalent to the existing `levelling-guide`, `openers`, etc.

**Comment by rexatlanti** [comment]
When I was trying this out locally, I could just generate a file by hand and it would populate as it should. If no file existed. Nothing happened. So worst case, we can deploy this change and see that we need to make files.

Deploying this change isn't breaking at least, should no files exist.

**Comment by 22a** [comment]
> Deploying this change isn't breaking at least, should no files exist.

Cool, happy to approve then once https://github.com/The-Balance-FFXIV/balance-static/pull/1453#discussion_r799913480 is fixed 👍

**Comment by rexatlanti** [comment]
Should be fixed now

**Review by 22a — APPROVED** [review]

**Comment by lailleo** [comment]
To clarify, @rexatlanti, this adds the ability to create these in the CMS/editor too, correct?

**Comment by rexatlanti** [comment]
This change _is_ specifically to add it to the eitor/CMS

**Pull request merged**